src/components/Board.js

```diff
@@ -30,15 +30,61 @@ const Board = (props) => {
 
   const cardComponents = () => {
     return cards.map((cardh) => {
-      console.log(cardh)
+      //console.log(cardh)
       return (
         <section key={cardh.card.id}>
-          <Card id={cardh.card.id} text={cardh.card.text} emoji={cardh.card.emoji} />
+          <Card id={cardh.card.id} onDeleteClick={onDeleteClick} text={cardh.card.text} emoji={cardh.card.emoji} />
         </section>
       );
     });
   };
 
+  //name of function onDeleteClick = delete a card, need to pass in id
```

Comment:
Huge moving part here, please look at steps 1-6 first and follow what happened between the board.js and card.js. Basically, following through on when clicking the delete button, that the id is accessible, then deletes it, and re-renders the page.
Then, notice the Notes A-C next that is telling you what's happening on a high level